engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -125,6 +131,10 @@ IR.Module translateModule(Tree module) {
               var t = translateModuleSymbol(def);
               bindings = cons(t, bindings);
             }
+            case Tree.ArgumentBlockApplication app -> {
+              var t = translateComment(app);
```

Comment:
An [argument block](https://github.com/enso-org/design/blob/wip/wd/enso-spec/epics/enso-spec-1.0/03.%20Code%20format%20and%20layout.md#argument-list-code-blocks) applies all of its arguments (`getArguments`) to its lhs (`getLhs`). It is not a type of comment...

Reply:
Right now about fifty files in our standard library fail to pass thru `TreeToIr` conversion because documentation comment is represented as `Tree.ArgumentBlockApplication` - I have to process it somehow. It would, of course help if:
```
    ## Measure the amount of time it takes to execute a given computation.
    
       Arguments:
       - act: The action to perform.
       - label: A name for the measurement.
       - iter_size: The number of runs per iteration.
       - num_iters: The number of iterations per measurement.
    
       > Example
         Measure a computation called "foo" with an iteration size of 2 and a number
         of iterations of 1.
    
             import Standard.Examples
             import Standard.Test.Bench
    
             example_measure =
                 Bench.measure Examples.get_boolean "foo" iter_size=2 num_iters=1
    measure : Any -> Text -> Integer -> Integer -> Nothing
    measure = ~act -> label -> iter_size -> num_iters ->
        42
```
wasn't represented as `Tree.ArgumentBlockApplication` - but it is. Until the Rust part delivers better result for documentation comments, converting `Tree.ArgumentBlockApplication` by `translateComment` does the least harm.

Reply:
I removed this diff in [466ec48](https://github.com/enso-org/enso/pull/3747/commits/466ec486388ad45adecaad6f6278bf80f84bc821)